.github/workflows/unit-test-backend.yml

```diff
@@ -28,6 +27,9 @@ jobs:
         with:
           go-version: 1.16.1
 
+      - name: Setup git auth
+        run: git config --global url.https://${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/.insteadOf https://github.com/
+
```

Comment:
Do we need this now once the repository is public?

Reply:
No, reverted.